.github/filters.yml

```diff
@@ -31,13 +31,14 @@ pytest:
   - '.github/workflows/pytest.yml' # ci
 
 npm-test:
-  - '{jsapp,test,webpack,static,scripts}/**/*.!(md|py|sh|bash)'               # frontend
-  - 'static/openapi/*'                                                        # API
-  - '{package*.json,patches/*.patch,scripts/copy_fonts.py}'                   # npm + postinstall
-  - '{tsconfig.json,.swcrc,.babelrc*,.browserslistrc}'                        # compilers
-  - '{.editorconfig,.stylelint*,.eslint*,coffeelint*}'                        # linters
-  - '.gitignore'                                                              # (can affect tools)
-  - '.github/workflows/{npm-test,chromatic}.yml'                              # ci
+  - '{jsapp,test,webpack,static,scripts}/**/*.!(md|py|sh|bash)' # frontend
+  - '.storybook/**/*.!(md)'                                     # storybook
```

Comment:
Adding `.storybook` to the `npm-test` filters ensures that npm-test will re-run on a commits that only changes a file in `.storybook`.

Not central to the strategy of this PR, but important for future storybook PR's, so I'm including it in this one.